src/store.ts

```diff
@@ -307,6 +307,10 @@ export class Store {
   }
 
   private static validateKey(key: string) {
+    if (key === '') {
+      throw new Error('Blob key must not be empty.')
```

Comment:
You could also make this part of the check on line 318 and adjust the message to something like:

> Blob key must be a sequence of Unicode characters whose UTF-8 encoding is between 1 and 600 bytes long.

Just an idea, no strong preference.

Reply:
That's nifty - but if I were debugging this exception, I think i'd be way more confused by the words "Unicode" and "UTF-8" than necessary. Gonna keep it like it is!